packages/@okta/vuepress-site/docs/reference/core-okta-api/index.md

```diff
@@ -26,30 +26,321 @@ REST endpoints to configure objects whenever you need. For example:
+### URL Namespace
 
-All URLs listed in the documentation should be preceded with your organization's subdomain (tenant) and API version: `https://{yourOktaDomain}.com/api/{apiversion}`
+All URLs listed in the documentation should be preceded with your organization's subdomain (tenant) and API version: `https://${yourOktaDomain}.com/api/${apiversion}`
 The API version is currently `v1`.
 
 > **Note:** All API requests must use the HTTPS scheme.
+
+### Media types
+
+> **Note:** JSON responses, including errors, may contain user input. To help prevent potential cross-site scripting attacks, make sure to properly escape all values before use in a browser or any HTML context.
+
+The API currently supports only JSON as an exchange format. Be sure to set both the `Content-Type` and `Accept` headers for every request as `application/json`.
+
+All Date objects are returned in [ISO 8601 format](https://tools.ietf.org/html/rfc3339):
+
+    YYYY-MM-DDTHH:mm:ss.SSSZ
+
+### Character sets
+
+Okta supports a subset of the `UTF-8` specification. Specifically, any character that can be encoded in three bytes or less is supported. BMP characters and supplementary characters that must be encoded using four bytes aren't supported at this time.
+
+### HTTP verbs
+
+Where possible, the Okta API strives to use appropriate HTTP verbs for each action.
+
+#### GET
+
+Used for retrieving objects
+
+#### POST
+
+Used for creating objects or performing custom actions (such as
+user lifecycle operations). For POST requests with no `body` param, set the `Content-Length` header to zero.
+
+#### PUT
+
+Used for replacing objects or collections. For PUT requests with no `body` param, set the `Content-Length` header to zero.
+
+#### DELETE
+
+Used for deleting objects
+
+> **Note:** Any PUT or POST request with no `Content-Length` header nor a body returns a 411 error. To get around this, include a `Content-Length: 0` header.
+
+### Client request context
+
+Okta derives client request context directly from the HTTP request headers and client TCP socket. Request context is used to evaluate policies such as Okta Sign-On Policy and provide client information for [troubleshooting and auditing](/docs/reference/api/events/#client-objecttype) purposes.
+
+### User Agent
+
+Okta supports the standard `User-Agent` HTTP header to identify the user's browser or application. Always send a `User-Agent` string to uniquely identify your client application and version, for example: `Oktaprise/1.1`.
+
+> **Note:** If your application is acting as a gateway or proxy, you should forward the `User-Agent` of the originating client with your API requests.
+
+#### Format a User-Agent string
+
+Make sure that the `User-Agent` string that your app constructs is in the correct format. This ensures that Okta can parse the `OS` and `Browser` fields. Good `User-Agent` strings that can be correctly parsed by Okta contain browser and system information, platform details, and any extensions.
```

Comment:
```suggestion
Ensure that the `User-Agent` string that your app constructs is in the correct format so that Okta can parse the `OS` and `Browser` fields. The `User-Agent` strings that can be correctly parsed by Okta contain browser and system information, platform details, and any extensions.
```